src/Form/AdminSettingsForm.php

```diff
@@ -68,7 +61,7 @@ public static function create(ContainerInterface $container) {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, Request $request = NULL) {
+  public function buildForm(array $form, FormStateInterface $form_state) {
```

Comment:
As a customer you do want to know what it will default to, otherwise black magic defaults will always (in my experience) come and haunt you for years. Also for us, it's very important we know at any given time what the defaults are. It's fine that we have defaults in the lift.js files but we should not start to rely on those. Certainly not because those are region specific. I would like us to reconsider the dropdown for at least 2 regions, where we then move everything else into an advanced form. The dropdown is equal to the defaults with the difference that users can actually go and check what the dropdown filled in for them. I am not convinced that is a follow-up we should delay or do "later". We have the story time to do it now, so why not go ahead and do it. It's not a massive change in functionality but will, imho, massively improve the user experience since they don't have to know all these values. They only have to configure their Lift Account and Site Identifier and select a region... For every region we add, we can bring out a new release or allow anyone to fill in the values in the advanced form.